src/main/java/com/notably/view/SideBarTreeViewImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.notably.view;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.TreeItem;
+import javafx.scene.control.TreeView;
+import javafx.scene.layout.Region;
+
+/**
+ *
+ */
+public class SideBarTreeViewImpl extends ViewPart<Region> {
+
+    private static final String FXML = "SideBarTreeView.fxml";
+
+    @FXML
+    private TreeView<String> sideBarTreeViewRoot;
+
+    public SideBarTreeViewImpl() {
+        super(FXML);
+        loadTreeItems();
+    }
+
+    /**
+     *
+     */
+    private void loadTreeItems() {
+        TreeItem<String> root = new TreeItem<>("Root");
+        root.setExpanded(true);
+        TreeItem<String> notably = new TreeItem<>("Notably");
+        notably.setExpanded(true);
+        root.getChildren().add(notably);
+        notably.getChildren().add(new TreeItem<>("Another Note"));
+        notably.getChildren().add(new TreeItem<>("CS2103"));
+        notably.getChildren().add(new TreeItem<>("ST2334"));
+
+        sideBarTreeViewRoot.setRoot(root);
+    }
```

Comment:
Haha interesting that you use `TreeItem` to render the sidebar too!

Currently, @firzanarmani is working on a PR (#125) that implements our block tree using instances of `TreeItem` too. Perhaps we can find a way to make these work together nicely.